model/fieldtypes/DBFloat.php

```diff
@@ -60,3 +60,7 @@ public function prepValueForDB($value) {
 	}
 
 }
+
+if (PHP_MAJOR_VERSION < 7) {
+	include_once 'compat/Float.php';
```

Comment:
One issue with Loz' patch is that calling class_exists('Int') or a before class_exists('DBInt') or similar will return false, as the manifest won't pick up Int. Ditto subclassing Int, etc.

If that's a showstopper, one solution would be to load an additional autoloader if PHP < 7, which would load the Int and Float classes as needed.

I'd prefer an additional autoloader to hacking manifest, simply because it reduces the impact on pre-existing code.